ocw/lib/emailnotify.py

```diff
@@ -65,26 +60,24 @@ def send_leftover_notification():
 
 
 def send_cluster_notification(namespace, clusters):
-    cfg = ConfigFile()
-    cfg_path = ['notify.cluster.namespace.{}'.format(namespace), 'to']
-    if not cfg.has('notify') or not cfg.has(cfg_path):
+    if not ConfigFile().has('notify'):
```

Comment:
@cfconrad  also check this

Reply:
fine for me. You could also  use:
```
 if not PCWConfig.get_feature_property('notify.cluster', 'to', namespace):
    return
```
wdyt?

Reply:
this would be change in the logic what we have here . No necessarily wrong but not "just code refactoring" anymore : 
1. currently in case you not define `[notify]` section in the config you will exit here 
2. in your version you would rely on totally different logic mixing several places in config file **and** also content of `default_values`  dictionary within a code 
for me current approach  **here** looks more straightforward , but I am open to discuss this . just wanted to make clear that it is not "same way just looks nicer "